### DISCUSSION TITLE
How to Remove LineCart start end padding

### DISCUSSION THREAD

**Discussion by YKW93** [patrykandpatrick/vico#596] [Questions] [answered]
### How to reproduce

How to Remove LineCart Start End Value Padding

Below is my code When I use compose vico I get padding? gaps? at the beginning and end and I want to get rid of them, how can I do that?

The result I want is the first image below.

~~~
    val list = listOf(
        FloatEntry(0f, 1f),
        FloatEntry(1f, 0f),
        FloatEntry(2f, 1f)
    )
    Chart(
        chart = lineChart(lines = chartLines),
        model = ChartEntryModelProducer(list).requireModel(),
        chartScrollSpec = rememberChartScrollSpec(isScrollEnabled = false),
        modifier = modifier
            .fillMaxWidth()
            .height(102.dp)
    )
~~~

### Observed behavior

vico library
![스크린샷 2024-03-04 오후 3 58 40](https://github.com/patrykandpatrick/vico/assets/41371709/9655cb8e-55c5-4c8f-84ca-0501a024ff57)

### Expected behavior

MPAndroidChart
![스크린샷 2024-03-04 오후 3 58 51](https://github.com/patrykandpatrick/vico/assets/41371709/f9e0d67b-fb6c-48a9-854d-33dece5dee9d)

### Vico version(s)

Latest stable version

### Android version(s)

14

### Additional information

using vico 1.13.1 version

**Comment by patrickmichalik** [comment, marked as answer]
Hello! You can use `HorizontalLayout.FullWidth` to remove this padding:

```kt
horizontalLayout = HorizontalLayout.fullWidth()
```

By the way, please note that the manner in which the chart data is currently handled could be simplified and made more performant as follows:

```kt
model = remember { entryModelOf(1, 0, 1) }
```

I also wanted to let you know that version 1.13.1 isn’t the latest stable version. Updating to version 1.14.0 would be highly recommended. Cheers!

**Reply by YKW93** [reply]
Thank you very much for your quick reply 😄

**Reply by patrickmichalik** [reply]
You’re welcome!

**Comment by danielpordel** [comment]
How can I remove start padding in the latest version (2.1.0)?
I couldn't find `HorizontalLayout`

**Reply by Gowsky** [reply]
Hello, @danielpordel. Please see the `addExtremeLabelPadding` parameter of `HorizontalAxis.ItemPlacer.aligned`.
